subprojects/files/src/test/groovy/org/gradle/api/internal/file/archive/impl/StreamZipInputTest.groovy

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.internal.file.archive.impl
+
+import org.gradle.test.fixtures.file.TestNameTestDirectoryProvider
+import org.junit.Rule
+import spock.lang.Specification
+
+import java.util.zip.ZipEntry
+import java.util.zip.ZipOutputStream
+
+
+class StreamZipInputTest extends Specification {
```

Comment:
Should matching (but opposing) tests be added to `FileZipInputTest`?

Reply:
Done.